scripts/gitlab/build-unix.sh

```diff
@@ -27,20 +27,18 @@ fi
 
 echo "_____ Post-processing binaries _____"
 rm -rf artifacts
-mkdir -p artifacts
-cd artifacts
-mkdir -p $CARGO_TARGET
-cd $CARGO_TARGET
+mkdir -p artifacts/$CARGO_TARGET
+
 if [ "${CARGO_TARGET}" = "armv7-linux-androideabi" ]
 then
 # only thing we need for android
- cp -v ../../target/$CARGO_TARGET/release/libparity.so ./libparity.so
+  cp -v target/$CARGO_TARGET/release/libparity.so ./libparity.so
```

Comment:
binaries have to go into the artifacts directory.
initially the script cd'ed into it now maybe
```
cp -v target/$CARGO_TARGET/release/libparity.so ./artifacts/libparity.so
```

Reply:
same below for `for binary in $(ls)`
should work with
```
for binary in $(ls artifacts)
```
but better test.

Reply:
thanks for pointing that out, fixed in next commit